platform-coop-toolkit/pinecone
# feat: updating form element states to match wires (fixes #240)

Update form element states to match designs. Fixes #240 .

This is work in progress.


## Description

This updates the states of form elements and buttons to match the updated designs.

Not addressed in this pull request (to be addressed in another pull request):

- Tags
- Links
- Lists
- Accordions
- Pagination

## Steps to test

1. Go to each of the components below and verify that default, hover, active, focus, disabled, and error states match the designs.

Buttons:
- [Default](https://deploy-preview-288--pinecone.netlify.com/components/detail/button--default.html)
- [Default inverted](https://deploy-preview-288--pinecone.netlify.com/components/detail/button--inverse.html)
- [Secondary](https://deploy-preview-288--pinecone.netlify.com/components/detail/button--secondary.html)
- [Secondary Inverted](https://deploy-preview-288--pinecone.netlify.com/components/detail/button--secondary-inverse.html)
- [Borderless default](https://deploy-preview-288--pinecone.netlify.com/components/detail/button--borderless.html)
- [Borderless inverted](https://deploy-preview-288--pinecone.netlify.com/components/detail/button--borderless-inverse.html)
- [Destructive borderless](https://deploy-preview-288--pinecone.netlify.com/components/detail/button--destructive-borderless.html)
- [Destructive inverted](https://deploy-preview-288--pinecone.netlify.com/components/detail/button--destructive-borderless-inverse.html)
- [Disc default](https://deploy-preview-288--pinecone.netlify.com/components/detail/button--disc.html)
- [Disc inverse](https://deploy-preview-288--pinecone.netlify.com/components/detail/button--disc-inverse.html)
- [Tag button](https://deploy-preview-288--pinecone.netlify.com/components/detail/button--tag-button.html)

Form elements:
- [Text Input](https://deploy-preview-288--pinecone.netlify.com/components/detail/inputs--default.html)
- [Text Input Inverted](https://deploy-preview-288--pinecone.netlify.com/components/detail/inputs--default_inverse.html)
- [Text Area](https://deploy-preview-288--pinecone.netlify.com/components/detail/textareas--default.html)
- [Text Area Inverted](https://deploy-preview-288--pinecone.netlify.com/components/detail/textareas--default-inverse.html)
- [Select](https://deploy-preview-288--pinecone.netlify.com/components/detail/selects--default.html)
- [Select Inverted](https://deploy-preview-288--pinecone.netlify.com/components/detail/selects--default-inverse.html)
- [Checkboxes](https://deploy-preview-288--pinecone.netlify.com/components/detail/checkboxes--default.html)
- [Checkboxes Inverted](https://deploy-preview-288--pinecone.netlify.com/components/detail/checkboxes--default-inverse.html)
- [Radio Buttons](https://deploy-preview-288--pinecone.netlify.com/components/detail/radio-buttons--default.html)
- [Radio Buttons Inverted](https://deploy-preview-288--pinecone.netlify.com/components/detail/radio-buttons--default-inverse.html)
- [Submit Button](https://deploy-preview-288--pinecone.netlify.com/components/detail/submit--default.html)
- [Submit Button Inverted](https://deploy-preview-288--pinecone.netlify.com/components/detail/submit--default-inverse.html)

**Expected behavior:** 

Different states should visually match the designs. See: 
[03 States Table.pdf](https://github.com/platform-coop-toolkit/pinecone/files/4388608/03.States.Table.pdf)


## Related issues

#240
